### PR TITLE
LW-11521 be-v0.30.1

### PR DESCRIPTION
**Pull request by iccicci**
# Context

The `asset-projector` has some performances issues.

# Proposed Solution

Added an index to reduce SQL queries weight.

# Important Changes Introduced

Minor query refactoring to remove a `JOIN`.

### CODE REVIEW

File: packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { NftMetadataEntity } from '@cardano-sdk/projection-typeorm';
+
+export class NftMetadataIndexMigration1726479765002 implements MigrationInterface {
+  static entity = NftMetadataEntity;
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```
Comment (mkazlauskas) [inline packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts:7]: Based on the query it looks like it's the other way around :sweat_smile: 
```suggestion
  public async down(queryRunner: QueryRunner): Promise<void> {
```

**Review by mkazlauskas — CHANGES_REQUESTED** [review]

File: packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts (hunk 1 above)
Comment (iccicci) [reply packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts:7]: 🤦

Comment (iccicci) [reply packages/cardano-services/src/Projection/migrations/1726479765002-nft-metadata-index.ts:7]: 05fc92cbbcb (already squashed) should solve this

**Review by mkazlauskas — APPROVED** [review]
Nice work! :rocket:

**Comment by gytis-ivaskevicius** [comment]
How are we releasing this? merging it to release branch? new release off master?

**Review by gytis-ivaskevicius — APPROVED** [review]

**Review by gytis-ivaskevicius — DISMISSED** [review]

**Review by mkazlauskas — APPROVED** [review]

**Review by mkazlauskas — DISMISSED** [review]

**Comment by iccicci** [comment]
Some problems with the workflow, makes this PR not approvable.

**Pull request closed without merging**